PennMobile/Events/EventsTableViewCell.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  EventsTableViewCell.swift
+//  PennMobile
+//
+//  Created by Samantha Su on 10/1/21.
+//  Copyright © 2021 PennLabs. All rights reserved.
+//
+
+import UIKit
+import Kingfisher
+import SwiftSoup
+
+class EventsTableViewCell: UITableViewCell {
+
+    lazy var imageExistsConstraint = eventImageView.widthAnchor.constraint(equalToConstant:120)
+    lazy var imageMissingConstraint = eventImageView.widthAnchor.constraint(equalToConstant:0)
+    var isExpanded = false
+
+    var pennEvent: PennEvents?{
+        didSet{
+            guard let event = pennEvent else {return}
+            if event.media_image == ""{
```

Comment:
We'll eventually add a linter, but for now lets try our best to keep coding conventions like spaces before `{`